ios/BT/ExposureManager.swift

```diff
@@ -525,4 +523,61 @@ private extension ExposureManager {
     localUncompressedURLs = []
     downloadedPackages = []
   }
+
+  func processExposureDetectionSummary(_ summary: ENExposureDetectionSummary?, completion: @escaping (([ExposureDetectionSummary]) -> Void)) {
+    if let summary = summary {
+
+      // Serialize exposure detection summary
+      let exposureDetectionSummary = ExposureDetectionSummary(summary)
+
+      // Prune exposure detection summaries >14 days from realm
+      BTSecureStorage.shared.pruneExposureDetectionSummaries()
+
+      // Fetch exposure summaries <14 days
+      let unusedExposureSummaries = BTSecureStorage.shared.allExposureDetectionSummaries()
+
+      // Store new exposure summary in realm
+      BTSecureStorage.shared.storeExposureDetectionSummary(exposureDetectionSummary)
+
+      // Call scoring server
+      apiClient.request(ExposureDetectionSummaryListRequest.post(exposureDetectionSummary,
+                                                                 unusedExposureSummaries),
+                        requestType: .scoring) { result in
+                          switch result {
+                          case .success(let scoringServerResponse):
+                            let summaries = scoringServerResponse.notifications.map { $0.exposureSummaries }.flatMap { $0 }
+                            completion(summaries)
+                          case .failure:
+                            completion([])
```

Comment:
I think this is outside of the scope of this change, but, how hard would it be to try the same thing again later? (Perhaps the next time we fetch a new key batch.)
(Consider writing a note here that we should handle that case.)

Reply:
ah - this piece is not related to fetching keys from the key server; it is the completion after the mobile client makes a request to the scoring server. If successful, if passes the `notification` objects returned from the server into the completion. If it fails due to a server error, we should log it, but the completion should still be executed

Reply:
No, that's what I mean - the part about the key server was just a suggestion of when might be a good time to retry. (We don't actually need retries to be instant, since the user isn't actually interacting here!)

But basically, if the server were down for some reason, it would be nice to retry a bit later. Could be done as a follow-up change though.